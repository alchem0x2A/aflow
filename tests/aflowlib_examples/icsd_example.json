{"aurl":"aflowlib.duke.edu:AFLOWDATA/ICSD_WEB/HEX/Ac1Br3_ICSD_31578","auid":"aflow:fc70fda6652015cb","title":"Ac1Br3_ICSD_31578","data_api":"aapi1.2","data_source":"aflowlib","loop":["bader","bands","lock","magnetic","thermodynamics"],"code":"vasp.4.6.35","compound":"Ac2Br6","prototype":"Ac1Br3_ICSD_31578","nspecies":2,"natoms":8,"natoms_orig":8,"composition":[2,6],"density":5.56878,"density_orig":5.86011,"scintillation_attenuation_length":1.46746,"stoichiometry":[0.25,0.75],"species":["Ac","Br"],"species_pp":["Ac","Br"],"dft_type":["PAW_PBE"],"species_pp_version":["Ac:PAW_PBE:06Sep2000","Br:PAW_PBE:06Sep2000"],"species_pp_ZVAL":[11,7],"valence_cell_iupac":48,"valence_cell_std":48,"volume_cell":278.353,"volume_atom":34.7942,"volume_cell_orig":264.515,"volume_atom_orig":33.0644,"pressure":0,"stress_tensor":[-0.37,0,0,0,-0.38,0,0,0,-0.81],"pressure_residual":-0.52,"Pulay_stress":0,"geometry":[8.247654,8.247618,4.725054,90,90,120],"geometry_orig":[8.07,8.07,4.69,90,90,120],"Egap":4.1084,"Egap_fit":6.45112,"Egap_type":"insulator-indirect","energy_cell":-34.8119,"energy_atom":-4.35149,"energy_cutoff":302.8,"delta_electronic_energy_convergence":2.44853e-07,"delta_electronic_energy_threshold":0.0001,"kpoints_relax":[8,8,11],"kpoints_static":[10,10,15],"kpoints_bands_path":["\\Gamma-M","M-K","K-\\Gamma","\\Gamma-A","A-L","L-H","H-A","L-M","K-H"],"kpoints_bands_nkpts":20,"kpoints":[[8,8,11],[10,10,15],["\\Gamma-M","M-K","K-\\Gamma","\\Gamma-A","A-L","L-H","H-A","L-M","K-H"],20],"enthalpy_cell":-34.8119,"enthalpy_atom":-4.35149,"eentropy_cell":0.00608044,"eentropy_atom":0.000760055,"enthalpy_formation_cell":-17.0842,"enthalpy_formation_atom":-2.13553,"entropic_temperature":44069.4,"PV_cell":0,"PV_atom":0,"spin_cell":0,"spin_atom":0,"spinD":[0,0,0,0,0,0,0,0],"spinF":0,"calculation_time":9666.65,"calculation_memory":1729.02,"calculation_cores":16,"nbondxx":[4.72505435,3.21010803,3.68978682],"sg":["P6_{3}/m #176","P6_{3}/m #176","P6_{3}/m #176"],"sg2":["P6_{3}/m #176","P6_{3}/m #176","P6_{3}/m #176"],"spacegroup_orig":176,"spacegroup_relax":176,"forces":[[-2.2e-05,-0.000398,0],[2.2e-05,0.000398,-0],[-0.00295,0.013167,0],[-0.009831,-0.009164,-0],[0.012868,-0.003941,0],[0.00295,-0.013167,0],[0.009831,0.009164,0],[-0.012868,0.003941,0]],"positions_cartesian":[[4.12382,2.38094,1.18126],[4.12382,-2.38096,3.54379],[2.84966,-0.6203,1.18126],[3.23616,-4.36473,1.18126],[6.28566,-2.15774,1.18126],[5.39798,0.62028,3.54379],[5.01148,4.36471,3.54379],[1.96198,2.15772,3.54379]],"positions_fractional":[[0.33332924,0.6666717,0.25],[0.66667076,0.3333283,0.75],[0.3889336,0.30208995,0.25],[0.69791153,0.086834274,0.25],[0.9131609,0.61107077,0.25],[0.6110664,0.69791005,0.75],[0.30208847,0.91316573,0.75],[0.086839098,0.38892923,0.75]],"Bravais_lattice_orig":"HEX","lattice_variation_orig":"HEX","lattice_system_orig":"hexagonal","Pearson_symbol_orig":"hP8","Bravais_lattice_relax":"HEX","lattice_variation_relax":"HEX","lattice_system_relax":"hexagonal","Pearson_symbol_relax":"hP8","crystal_family_orig":"hexagonal","crystal_system_orig":"hexagonal","crystal_class_orig":"hexagonal-dipyramidal","point_group_Hermann_Mauguin_orig":"6/m","point_group_Schoenflies_orig":"C_6h","point_group_orbifold_orig":"6*","point_group_type_orig":"centrosymmetric","point_group_order_orig":12,"point_group_structure_orig":"2_Ã—_cyclic","Bravais_lattice_lattice_type_orig":"HEX","Bravais_lattice_lattice_variation_type_orig":"HEX","Bravais_lattice_lattice_system_orig":"hexagonal","Bravais_superlattice_lattice_type_orig":"HEX","Bravais_superlattice_lattice_variation_type_orig":"HEX","Bravais_superlattice_lattice_system_orig":"hexagonal","Pearson_symbol_superlattice_orig":"hP8","reciprocal_geometry_orig":[0.89903,0.89903,1.3397,90,90,60],"reciprocal_volume_cell_orig":0.937754,"reciprocal_lattice_type_orig":"HEX","reciprocal_lattice_variation_type_orig":"HEX","Wyckoff_letters_orig":[["c"],["h"]],"Wyckoff_multiplicities_orig":[[2],[6]],"Wyckoff_site_symmetries_orig":[["-6.."],["m.."]],"crystal_family":"hexagonal","crystal_system":"hexagonal","crystal_class":"hexagonal-dipyramidal","point_group_Hermann_Mauguin":"6/m","point_group_Schoenflies":"C_6h","point_group_orbifold":"6*","point_group_type":"centrosymmetric","point_group_order":12,"point_group_structure":"2_Ã—_cyclic","Bravais_lattice_lattice_type":"HEX","Bravais_lattice_lattice_variation_type":"HEX","Bravais_lattice_lattice_system":"hexagonal","Bravais_superlattice_lattice_type":"HEX","Bravais_superlattice_lattice_variation_type":"HEX","Bravais_superlattice_lattice_system":"hexagonal","Pearson_symbol_superlattice":"hP8","reciprocal_geometry":[0.87967,0.87967,1.3298,90,90,60],"reciprocal_volume_cell":0.891134,"reciprocal_lattice_type":"HEX","reciprocal_lattice_variation_type":"HEX","Wyckoff_letters":[["c"],["h"]],"Wyckoff_multiplicities":[[2],[6]],"Wyckoff_site_symmetries":[["-6.."],["m.."]],"anrl_label_orig":"AB3_hP8_176_c_h","anrl_parameter_list_orig":["a","c/a","x2","y2"],"anrl_parameter_values_orig":[8.07,0.581165,0.917067,0.625033],"anrl_label_relax":"AB3_hP8_176_c_h","anrl_parameter_list_relax":["a","c/a","x2","y2"],"anrl_parameter_values_relax":[8.2477,0.572899,0.388938,0.302085],"bader_net_charges":[2.1004,2.1004,-0.7004,-0.7004,-0.6991,-0.6999,-0.7008,-0.7001],"bader_atomic_volumes":[24.92,24.92,38.09,38.11,38.05,38.08,38.12,38.06],"files":["AECCAR0.static.xz","AECCAR2.static.xz","Ac1Br3_ICSD_31578.cif","Ac1Br3_ICSD_31578.png","Ac1Br3_ICSD_31578_BZ.png","Ac1Br3_ICSD_31578_Bader_10_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_10_Br.jvxl","Ac1Br3_ICSD_31578_Bader_20_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_20_Br.jvxl","Ac1Br3_ICSD_31578_Bader_25_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_25_Br.jvxl","Ac1Br3_ICSD_31578_Bader_30_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_30_Br.jvxl","Ac1Br3_ICSD_31578_Bader_33_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_33_Br.jvxl","Ac1Br3_ICSD_31578_Bader_40_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_40_Br.jvxl","Ac1Br3_ICSD_31578_Bader_50_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_50_Br.jvxl","Ac1Br3_ICSD_31578_Bader_60_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_60_Br.jvxl","Ac1Br3_ICSD_31578_Bader_67_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_67_Br.jvxl","Ac1Br3_ICSD_31578_Bader_75_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_75_Br.jvxl","Ac1Br3_ICSD_31578_Bader_80_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_80_Br.jvxl","Ac1Br3_ICSD_31578_Bader_90_Ac.jvxl","Ac1Br3_ICSD_31578_Bader_90_Br.jvxl","Ac1Br3_ICSD_31578_DOS.png","Ac1Br3_ICSD_31578_PEDOS_1_8_Ac.png","Ac1Br3_ICSD_31578_PEDOS_3_8_Br.png","Ac1Br3_ICSD_31578_abader.out","Ac1Br3_ICSD_31578_bandsdata.json.xz","Ac1Br3_ICSD_31578_corner.cif","Ac1Br3_ICSD_31578_dosdata.json.xz","Ac1Br3_ICSD_31578_sconv.cif","Ac1Br3_ICSD_31578_sconv_corner.cif","Ac1Br3_ICSD_31578_sprim.cif","Ac1Br3_ICSD_31578_sprim_corner.cif","Ac1Br3_ICSD_31578_structure_relax.json","Ac1Br3_ICSD_31578_structure_relax1.json","CHGCAR.static.xz","CONTCAR.relax","CONTCAR.relax.abinit","CONTCAR.relax.aims","CONTCAR.relax.qe","CONTCAR.relax.vasp","CONTCAR.relax1","DOSCAR.static.xz","EIGENVAL.bands.xz","INCAR.bands","KPOINTS.bands","KPOINTS.relax","KPOINTS.static","OUTCAR.static.xz","POSCAR.bands","aflow.agroup.bands.json.xz","aflow.agroup.bands.out.xz","aflow.agroup.orig.json.xz","aflow.agroup.orig.out.xz","aflow.agroup.relax.json.xz","aflow.agroup.relax.out.xz","aflow.fgroup.bands.json.xz","aflow.fgroup.bands.out.xz","aflow.fgroup.orig.json.xz","aflow.fgroup.orig.out.xz","aflow.fgroup.relax.json.xz","aflow.fgroup.relax.out.xz","aflow.iatoms.bands.json.xz","aflow.iatoms.bands.out.xz","aflow.iatoms.orig.json.xz","aflow.iatoms.orig.out.xz","aflow.iatoms.relax.json.xz","aflow.iatoms.relax.out.xz","aflow.pgroup.bands.json.xz","aflow.pgroup.bands.out.xz","aflow.pgroup.orig.json.xz","aflow.pgroup.orig.out.xz","aflow.pgroup.relax.json.xz","aflow.pgroup.relax.out.xz","aflow.pgroup_xtal.bands.json.xz","aflow.pgroup_xtal.bands.out.xz","aflow.pgroup_xtal.orig.json.xz","aflow.pgroup_xtal.orig.out.xz","aflow.pgroup_xtal.relax.json.xz","aflow.pgroup_xtal.relax.out.xz","aflow.pgroupk.bands.json.xz","aflow.pgroupk.bands.out.xz","aflow.pgroupk.orig.json.xz","aflow.pgroupk.orig.out.xz","aflow.pgroupk.relax.json.xz","aflow.pgroupk.relax.out.xz","aflow.pgroupk_xtal.bands.json.xz","aflow.pgroupk_xtal.bands.out.xz","aflow.pgroupk_xtal.orig.json.xz","aflow.pgroupk_xtal.orig.out.xz","aflow.pgroupk_xtal.relax.json.xz","aflow.pgroupk_xtal.relax.out.xz","aflow:fc70fda6652015cb.json","aflow:fc70fda6652015cb.out","aflowlib.json","aflowlib.out","edata.bands.json","edata.bands.out","edata.orig.json","edata.orig.out","edata.relax.json","edata.relax.out","index.php"],"node_CPU_Model":"AMD_Opteron(tm)_Processor_6378___","node_CPU_Cores":64,"node_CPU_MHz":1400,"node_RAM_GB":253,"aflow_version":"aflow30831","catalog":"ICSD","aflowlib_version":"3.1.221","aflowlib_date":"20190402_11:59:36_GMT-5"}
