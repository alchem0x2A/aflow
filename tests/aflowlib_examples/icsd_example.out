aurl=aflowlib.duke.edu:AFLOWDATA/ICSD_WEB/MCLC/Ag1As1F6Se3_ICSD_418700 | auid=aflow:6cd78dd3c73693e6 | data_api=aapi1.2 | title=Ag1As1F6Se3_ICSD_418700 | data_source=aflowlib | loop=bader,bands,lock,magnetic,thermodynamics | code=vasp.4.6.35 | compound=Ag2As2F12Se6 | prototype=Ag1As1F6Se3_ICSD_418700 | nspecies=4 | natoms=22 | natoms_orig=22 | composition=2,2,12,6 | density=4.24646 | density_orig=4.66324 | scintillation_attenuation_length=2.91128 | stoichiometry=0.0909090909,0.0909090909,0.545454545,0.272727273 | species=Ag,As,F,Se | species_pp=Ag,As,F,Se | dft_type=PAW_PBE | species_pp_version=Ag:PAW_PBE:06Sep2000,As:PAW_PBE:06Sep2000,F:PAW_PBE:08Apr2002,Se:PAW_PBE:06Sep2000 | species_pp_ZVAL=11,5,7,6 | ldau_type=2 | ldau_l=2,0,0,0 | ldau_u=5.8,0,0,0 | ldau_j=0,0,0,0 | ldau_TLUJ=2;2,0,0,0;5.8,0,0,0;0,0,0,0 | valence_cell_iupac=66 | valence_cell_std=152 | volume_cell=417.366 | volume_atom=18.9712 | volume_cell_orig=380.063 | volume_atom_orig=17.2756 | pressure=0 | stress_tensor=1.4,0,0,0,1.48,0.05,0,0.05,1.42 | pressure_residual=1.43 | Pulay_stress=0 | geometry=9.352448,9.352448,5.410036,88.18888,88.18888,117.9134 | geometry_orig=9.030002,9.030002,5.29,88.76137,88.76137,118.1302 | Egap=1.8648 | Egap_fit=3.42675 | Egap_type=insulator-direct | energy_cell=-85.4901 | energy_atom=-3.88592 | energy_cutoff=560 | delta_electronic_energy_convergence=9.63548e-05 | delta_electronic_energy_threshold=0.0001 | kpoints_relax=7,7,10 | kpoints_static=7,7,10 | kpoints_bands_path=\Gamma-Y,Y-F,F-H,H-Z,Z-I,I-X,X-\Gamma,\Gamma-Z,M-\Gamma,\Gamma-N,X-Y_1,Y_1-H_1,I-F_1 | kpoints_bands_nkpts=20 | kpoints=7,7,10;7,7,10;\Gamma-Y,Y-F,F-H,H-Z,Z-I,I-X,X-\Gamma,\Gamma-Z,M-\Gamma,\Gamma-N,X-Y_1,Y_1-H_1,I-F_1;20 | enthalpy_cell=-85.4901 | enthalpy_atom=-3.88592 | eentropy_cell=0.0031457 | eentropy_atom=0.000142986 | PV_cell=0 | PV_atom=0 | spin_cell=0 | spin_atom=0 | spinD=0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 | spinF=0 | stoich=0.0909   0.0909   0.5455   0.2727 | calculation_time=66097.2 | calculation_memory=8358.17 | calculation_cores=32 | nbondxx=5.3832,3.8742,2.6873,2.8191,5.41,1.7622,4.1359,2.4926,3.2296,2.3635 | sg=C2/m #12,C2/m #12,C2/m #12 | sg2=C2/m #12,C2/m #12,C2/m #12 | spacegroup_orig=12 | spacegroup_relax=12 | forces=0,-0,0;0,0,0;-0.022179,0,0;0.022179,0,0;-0.02049,2.2e-05,-0.012177;-0.02049,-2.2e-05,0.012177;0.02049,-2.2e-05,0.012177;0.02049,2.2e-05,-0.012177;-0.004723,-0.016032,-0.003268;-0.004723,0.016032,0.003268;0.004723,0.016032,0.003268;0.004723,-0.016032,-0.003268;0.007252,-0.007608,0.014092;0.007252,0.007608,-0.014092;-0.007252,0.007608,-0.014092;-0.007252,-0.007608,0.014092;0,0.003211,-0.006841;0,-0.003211,0.006841;-0.002953,0.003541,-0.001589;-0.002953,-0.003541,0.001589;0.002953,-0.003541,0.001589;0.002953,0.003541,-0.001589 | positions_cartesian=0,4.82272,0.04447;-0,0.14088,2.70135;-5.23945,4.82272,0.04447;5.23945,4.82272,0.04447;1.49111,8.85684,4.82358;1.49111,1.07036,0.66806;-1.49111,1.07036,0.66806;-1.49111,8.85684,4.82358;-5.30028,3.88688,1.54738;-5.30028,6.04032,3.94426;5.30028,6.04032,3.94426;5.30028,3.88688,1.54738;-3.99365,5.87549,0.71152;-3.99365,4.05171,4.78012;3.99365,4.05171,4.78012;3.99365,5.87549,0.71152;0,7.04393,2.13711;0,2.88327,3.35453;1.8332,6.01503,3.28525;1.8332,3.91217,2.20639;-1.8332,3.91217,2.20639;-1.8332,6.01503,3.28525 | positions_fractional=0.5,0.5,0;-0,0,0.5;0.17306467,0.82693533,0;0.82693533,0.17306467,-0;0.98563282,0.79954636,0.87811542;0.20045364,0.01436718,0.12188458;0.01436718,0.20045364,0.12188458;0.79954636,0.98563282,0.87811542;0.064067748,0.72553048,0.27990929;0.27446952,0.93593225,0.72009071;0.93593225,0.27446952,0.72009071;0.72553048,0.064067748,0.27990929;0.35639134,0.85479013,0.12172771;0.14520987,0.64360866,0.87827229;0.64360866,0.14520987,0.87827229;0.85479013,0.35639134,0.12172771;0.71907651,0.71907651,0.3837262;0.28092349,0.28092349,0.6162738;0.72053179,0.49175312,0.59809781;0.50824688,0.27946821,0.40190219;0.27946821,0.50824688,0.40190219;0.49175312,0.72053179,0.59809781 | Bravais_lattice_orig=MCLC | lattice_variation_orig=MCLC3 | lattice_system_orig=monoclinic | Pearson_symbol_orig=mS44 | Bravais_lattice_relax=MCLC | lattice_variation_relax=MCLC3 | lattice_system_relax=monoclinic | Pearson_symbol_relax=mS44 | crystal_family_orig=monoclinic | crystal_system_orig=monoclinic | crystal_class_orig=monoclinic-prismatic | point_group_Hermann_Mauguin_orig=2/m | point_group_Schoenflies_orig=C_2h | point_group_orbifold_orig=2* | point_group_type_orig=centrosymmetric | point_group_order_orig=4 | point_group_structure_orig=2_x_cyclic | Bravais_lattice_lattice_type_orig=MCLC | Bravais_lattice_lattice_variation_type_orig=MCLC3 | Bravais_lattice_lattice_system_orig=monoclinic | Bravais_superlattice_lattice_type_orig=MCLC | Bravais_superlattice_lattice_variation_type_orig=MCLC3 | Bravais_superlattice_lattice_system_orig=monoclinic | Pearson_symbol_superlattice_orig=mS44 | reciprocal_geometry_orig=0.78953,0.78953,1.1888,92.068,92.068,61.825 | reciprocal_volume_cell_orig=0.652655 | reciprocal_lattice_type_orig=MCLC | reciprocal_lattice_variation_type_orig=MCLC1 | Wyckoff_letters_orig=b,c;g;j,j,j;i,j | Wyckoff_multiplicities_orig=2,2;4;8,8,8;4,8 | Wyckoff_site_symmetries_orig=2/m,2/m;2;1,1,1;m,1 | crystal_family=monoclinic | crystal_system=monoclinic | crystal_class=monoclinic-prismatic | point_group_Hermann_Mauguin=2/m | point_group_Schoenflies=C_2h | point_group_orbifold=2* | point_group_type=centrosymmetric | point_group_order=4 | point_group_structure=2_x_cyclic | Bravais_lattice_lattice_type=MCLC | Bravais_lattice_lattice_variation_type=MCLC3 | Bravais_lattice_lattice_system=monoclinic | Bravais_superlattice_lattice_type=MCLC | Bravais_superlattice_lattice_variation_type=MCLC3 | Bravais_superlattice_lattice_system=monoclinic | Pearson_symbol_superlattice=mS44 | reciprocal_geometry=0.76133,0.76133,1.1636,93.011,93.011,61.991 | reciprocal_volume_cell=0.594323 | reciprocal_lattice_type=MCLC | reciprocal_lattice_variation_type=MCLC1 | Wyckoff_letters=b,c;g;j,j,j;i,j | Wyckoff_multiplicities=2,2;4;8,8,8;4,8 | Wyckoff_site_symmetries=2/m,2/m;2;1,1,1;m,1 | anrl_label_orig=ABC6D3_mC44_12_bc_g_3j_ij | anrl_parameter_list_orig=a,b/a,c/a,beta,y3,x4,z4,x5,y5,z5,x6,y6,z6,x7,y7,z7,x8,y8,z8 | anrl_parameter_values_orig=9.284,1.66857,0.569798,87.59,0.82585,0.27884,0.618,0.1104,0.9049,0.1132,0.5993,0.3289,0.7201,0.3921,0.2492,0.8828,0.39042,0.8834,0.39885 | anrl_label_relax=ABC6D3_mC44_12_bc_g_3j_ij | anrl_parameter_list_relax=a,b/a,c/a,beta,y3,x4,z4,x5,y5,z5,x6,y6,z6,x7,y7,z7,x8,y8,z8 | anrl_parameter_values_relax=9.6458,1.66144,0.560866,86.4863,0.826935,0.280923,0.616274,0.10741,0.906957,0.121885,0.605201,0.330731,0.720091,0.394409,0.249199,0.878272,0.393858,0.885611,0.401902 | bader_net_charges=0.3955,0.6437,2.8951,2.8951,-0.6517,-0.6517,-0.6517,-0.6517,-0.6367,-0.6368,-0.6367,-0.6368,-0.6311,-0.6311,-0.6312,-0.6312,0.0794,0.0793,0.1709,0.1742,0.1638,0.1813 | bader_atomic_volumes=23.9380,21.3163,5.9795,5.9795,15.3208,15.3208,15.3208,15.3208,16.1827,16.1910,16.1827,16.1910,16.8755,16.8755,16.8776,16.8776,26.2509,26.2516,28.5346,28.5200,28.5604,28.4957 | files=AECCAR0.static.xz,AECCAR2.static.xz,Ag1As1F6Se3_ICSD_418700.cif,Ag1As1F6Se3_ICSD_418700.png,Ag1As1F6Se3_ICSD_418700_BZ.png,Ag1As1F6Se3_ICSD_418700_Bader_10_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_10_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_10_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_10_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_20_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_20_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_20_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_20_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_25_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_25_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_25_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_25_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_30_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_30_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_30_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_30_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_33_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_33_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_33_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_33_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_40_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_40_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_40_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_40_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_50_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_50_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_50_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_50_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_60_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_60_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_60_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_60_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_67_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_67_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_67_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_67_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_75_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_75_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_75_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_75_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_80_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_80_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_80_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_80_Se.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_90_Ag.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_90_As.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_90_F.jvxl,Ag1As1F6Se3_ICSD_418700_Bader_90_Se.jvxl,Ag1As1F6Se3_ICSD_418700_DOS.png,Ag1As1F6Se3_ICSD_418700_PEDOS_01_22_Ag.png,Ag1As1F6Se3_ICSD_418700_PEDOS_02_22_Ag.png,Ag1As1F6Se3_ICSD_418700_PEDOS_03_22_As.png,Ag1As1F6Se3_ICSD_418700_PEDOS_05_22_F.png,Ag1As1F6Se3_ICSD_418700_PEDOS_09_22_F.png,Ag1As1F6Se3_ICSD_418700_PEDOS_13_22_F.png,Ag1As1F6Se3_ICSD_418700_PEDOS_17_22_Se.png,Ag1As1F6Se3_ICSD_418700_PEDOS_19_22_Se.png,Ag1As1F6Se3_ICSD_418700_abader.out,Ag1As1F6Se3_ICSD_418700_bandsdata.json.xz,Ag1As1F6Se3_ICSD_418700_corner.cif,Ag1As1F6Se3_ICSD_418700_dosdata.json.xz,Ag1As1F6Se3_ICSD_418700_sconv.cif,Ag1As1F6Se3_ICSD_418700_sconv_corner.cif,Ag1As1F6Se3_ICSD_418700_sprim.cif,Ag1As1F6Se3_ICSD_418700_sprim_corner.cif,Ag1As1F6Se3_ICSD_418700_structure_relax.json,Ag1As1F6Se3_ICSD_418700_structure_relax1.json,CHGCAR.static.xz,CONTCAR.relax,CONTCAR.relax.abinit,CONTCAR.relax.aims,CONTCAR.relax.qe,CONTCAR.relax.vasp,CONTCAR.relax1,DOSCAR.static.xz,EIGENVAL.bands.xz,INCAR.bands,KPOINTS.bands,KPOINTS.relax,KPOINTS.static,OUTCAR.static.xz,POSCAR.bands,aflow.agroup.bands.json.xz,aflow.agroup.bands.out.xz,aflow.agroup.orig.json.xz,aflow.agroup.orig.out.xz,aflow.agroup.relax.json.xz,aflow.agroup.relax.out.xz,aflow.fgroup.bands.json.xz,aflow.fgroup.bands.out.xz,aflow.fgroup.orig.json.xz,aflow.fgroup.orig.out.xz,aflow.fgroup.relax.json.xz,aflow.fgroup.relax.out.xz,aflow.iatoms.bands.json.xz,aflow.iatoms.bands.out.xz,aflow.iatoms.orig.json.xz,aflow.iatoms.orig.out.xz,aflow.iatoms.relax.json.xz,aflow.iatoms.relax.out.xz,aflow.pgroup.bands.json.xz,aflow.pgroup.bands.out.xz,aflow.pgroup.orig.json.xz,aflow.pgroup.orig.out.xz,aflow.pgroup.relax.json.xz,aflow.pgroup.relax.out.xz,aflow.pgroup_xtal.bands.json.xz,aflow.pgroup_xtal.bands.out.xz,aflow.pgroup_xtal.orig.json.xz,aflow.pgroup_xtal.orig.out.xz,aflow.pgroup_xtal.relax.json.xz,aflow.pgroup_xtal.relax.out.xz,aflow.pgroupk.bands.json.xz,aflow.pgroupk.bands.out.xz,aflow.pgroupk.orig.json.xz,aflow.pgroupk.orig.out.xz,aflow.pgroupk.relax.json.xz,aflow.pgroupk.relax.out.xz,aflow.pgroupk_xtal.bands.json.xz,aflow.pgroupk_xtal.bands.out.xz,aflow.pgroupk_xtal.orig.json.xz,aflow.pgroupk_xtal.orig.out.xz,aflow.pgroupk_xtal.relax.json.xz,aflow.pgroupk_xtal.relax.out.xz,aflow:6cd78dd3c73693e6.json,aflow:6cd78dd3c73693e6.out,aflowlib.json,aflowlib.out,edata.bands.json,edata.bands.out,edata.orig.json,edata.orig.out,edata.relax.json,edata.relax.out,index.php | node_CPU_Model=AMD_Opteron(tm)_Processor_6378___ | node_CPU_Cores=64 | node_CPU_MHz=1400 | node_RAM_GB=253 | aflow_version=aflow31020 | catalog=ICSD | aflowlib_version=3.1.221 | aflowlib_date=20190328_11:12:01_GMT-5